qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -154,13 +154,13 @@ class Kubernetes implements OrchestratorMain {
             client.namespaces().create(namespace)
             defaultPspForNamespace(ns)
             provisionDefaultServiceAccount(ns)
-            log.debug "Created namespace ${ns}"
         } catch (KubernetesClientException kce) {
-            // 409 is already exists
             if (kce.code != 409) {
                 throw kce
             }
+            log.debug("Namespace ${ns} already exists", kce)
         }
+        log.info "Created namespace ${ns}"
```

Comment:
I find the log output misleading:

```
    17:39:44 | DEBUG | o.Kubernetes | Namespace qa already exists
    io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: POST at: https://35.184.16.207/api/v1/namespaces. Message: namespaces "qa" already exists. Received status: Status(apiVersion=v1, code=409, details=StatusDetails(causes=[], group=null, kind=namespaces, name=qa, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=namespaces "qa" already exists, metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=AlreadyExists, status=Failure, additionalProperties={}).
    	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.requestFailure(OperationSupport.java:682)
    	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.requestFailure(OperationSupport.java:661)
    	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.assertResponseCode(OperationSupport.java:612)
    	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:555)
    	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:518)
    	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleCreate(OperationSupport.java:305)
    	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:644)
    	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:83)
    	at io.fabric8.kubernetes.client.dsl.base.CreateOnlyResourceOperation.create(CreateOnlyResourceOperation.java:61)
    	at io.fabric8.kubernetes.client.dsl.InOutCreateable$create.call(Unknown Source)
    	at orchestratormanager.Kubernetes.ensureNamespaceExists(Kubernetes.groovy:154)
    	at orchestratormanager.OrchestratorMain$ensureNamespaceExists$8.callCurrent(Unknown Source)
    	at orchestratormanager.Kubernetes.setup(Kubernetes.groovy:167)
    	at orchestratormanager.OrchestratorMain$setup$7.call(Unknown Source)
    	at BaseSpecification.setupSpec(BaseSpecification.groovy:196)
    	at util.OnFailureInterceptor.intercept(OnFailure.groovy:72) [8 skipped]
     [43 skipped]
    17:39:44 | INFO  | o.Kubernetes | Created namespace qa
```

Maybe move this after the create()?

Reply:
Fixed